docs/file-rules-kts.md

```diff
@@ -0,0 +1,20 @@
+# The `rules.kts` file
+
+The `rules.kts` file allows defining custom policy rules to automatically reviews scan findings and using a rules DSL based on Kotlin.
+
+For each policy violation, you can define follow-up actions to help users resolve violations by themselves.
```

Comment:
What does that mean? How do you define follow-up actions?

Reply:
Refer to How To Fix - update the sentence to make this more clear